src/Core/View.js

```diff
@@ -458,4 +458,51 @@ View.prototype.execFrameRequesters = function execFrameRequesters(when, dt, upda
     }
 };
 
+const _eventCoords = new Vector2();
+/**
+ * Extract window coordinates from a mouse-event / touch-event
+ * @param {event} event - event can be a MouseEvent or a TouchEvent
+ * @param {number} touchIdx - finger index when using a TouchEvent (default: 0)
+ * @return {THREE.Vector2} - window coordinates (in pixels, 0-0 = top-left of the View)
```

Comment:
> Window Coordinates is the usual name...

in which context? Because for the web devs, this will be very confusing... I'd prefer `viewport` (a bit better, though there *is* a viewport size in a webpage) or `canvasCoords`? WDYT?

Reply:
I renamed `WindowCoords` to `ViewCoords`.